src/design-system/molecules/MarketDetailCard/MarketDetailCard.tsx

```diff
@@ -77,7 +77,7 @@ export const MarketDetailCard: React.FC<MarketDetailCardProps> = ({ title, items
       >
         {titleHeader}
       </AccordionSummary>
-      <AccordionDetails>{bodyContent}</AccordionDetails>
+      <AccordionDetails sx={{ paddingTop: '0.25rem' }}>{bodyContent}</AccordionDetails>
```

Comment:
use this from a constant

Reply:
i was thinking sx behaved more like mui props so it wouldn't rerender but this is fine I've moved this now.